README.md

```diff
@@ -28,56 +28,30 @@ At time of writing, it also serves the priority campaign pages. See the [Campaig
 
 ## Technical documentation
 
-This is a public facing Ruby on Rails application that retrieves browse content from APIs and presents it.
+This is a Ruby on Rails app, and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html).
 
-### Running the application
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) or the local `startup.sh` script to run the app. Read the [guidance on local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) to find out more about each approach, before you get started.
 
-- #### With startup scripts
+If you are using GOV.UK Docker, remember to combine it with the commands that follow. See the [GOV.UK Docker usage instructions](https://github.com/alphagov/govuk-docker#usage) for examples.
 
-```
-./startup.sh
-```
-
-The app should start on http://localhost:3070
+### Running the test suite
 
 ```
-./startup.sh --live
+bundle exec rake
```

Comment:
It's been mentioned

> In future, each frontend app could link to a single doc about their shared architecture, as that would be much more helpful than learning about it in each repo in isolation.

Wondering should this not apply to this testing snippet as well? 
What differentiates testing command snippets within the `README` whereas start-up commands direct to docs?